common/structs.py

```diff
@@ -463,6 +463,14 @@ def __init__(self, *args, **kwargs):
 ])
 
 
+ProxyType = IntEnumHack("ProxyType", [
+    ("none",    (1, {"display": "None",     "protocol": ""})),
+    ("socks4",  (2, {"display": "SOCKS4",   "protocol": "socks4"})),
+    ("socks5",  (3, {"display": "SOCKS5",   "protocol": "socks5"})),
+    ("http",    (4, {"display": "HTTP",     "protocol": "http"})),
```

Comment:
i dont see protocol parameter being used?

also i prefer having PascalCase for enum members, tags are an exception due to following forum standard, and i know some of the other existing enums are lowercase but im trying to keep to some standards from now on for consistency

Reply:
nevermind, using PascalCase here gets a syntax error due to None keyboard xD